integrations/va-auth0/src/index.tsx

```diff
@@ -22,6 +22,7 @@ type Auth0SiteInstallationConfiguration = {
     client_id?: string;
     issuer_base_url?: string;
     client_secret?: string;
+    include_claims_in_va_token?: boolean;
```

Comment:
Should we name this "enrich_session" or something simpler? the current name is very low level